components/messenger.rst

```diff
@@ -38,11 +38,28 @@ Concepts
    something can be a message broker or a third party API for example.
```

Comment:
Would be great if someone could edit the picture above in order to reflect Envelope usage inside the bus itself IMHO